README.md

```diff
@@ -453,6 +453,10 @@ And one topic for each additional bucket:
 
 Reference application where a configuration can be basically copied from is `demoapp`.
 
+To be able to see a task payload in production you have to configure which Role is able to do it. Any user with that role will be
+able to see plain payload, but the user and what data they queried will be logged.
+- `tasks-management.view-task-data-roles`
```

Comment:
`tw-tasks.core.tasks-management.view-task-data-roles` ?

and actually, it would be very helpful to provide an example, so that it's obvious that it's a list...

so it's roles, plural...